### PR TITLE
Change User libs to to use var path (non-system-specific path in classpath)

### LINKED ISSUE #64: User libraries should use relative paths in .classpath
When the classpath is updated to add user libraries, absolute paths are used, which will cause issues if the classpath is kept in version control and used on different computers with different user names.

### PR DESCRIPTION
**Pull request by Kevin-OConnor**
Closes #64

### CODE REVIEW

File: edu.wpi.first.wpilib.plugins.java/src/main/java/edu/wpi/first/wpilib/plugins/java/WPILibJavaPlugin.java [hunk 1]
```diff
@@ -198,7 +199,7 @@ public void updateVariables(IProject project) throws CoreException {
 					String fileNameSplit[] = file.getName().split("[.]");
 					if(fileNameSplit[fileNameSplit.length-1].equalsIgnoreCase("jar"))
 					{
-						IPath filePath = new Path(file.getAbsolutePath());
+						IPath filePath = new Path("USERLIBS_DIR/" + file.getName());
```
Comment (333fred) [inline edu.wpi.first.wpilib.plugins.java/src/main/java/edu/wpi/first/wpilib/plugins/java/WPILibJavaPlugin.java:202]: This doesn't have to be some variable syntax?

Comment (Kevin-OConnor) [reply edu.wpi.first.wpilib.plugins.java/src/main/java/edu/wpi/first/wpilib/plugins/java/WPILibJavaPlugin.java:202]: You mean like $ or {} or something? It does not and from what I can find should not be. Which is a little horrifying.

Comment (333fred) [reply edu.wpi.first.wpilib.plugins.java/src/main/java/edu/wpi/first/wpilib/plugins/java/WPILibJavaPlugin.java:202]: Yeah, that's what I meant. So it's like C++ macros. That's just wonderful.

**Review by 333fred — APPROVED** [review]

File: edu.wpi.first.wpilib.plugins.java/src/main/java/edu/wpi/first/wpilib/plugins/java/WPILibJavaPlugin.java (hunk 1 above)
Comment (PeterJohnson) [reply edu.wpi.first.wpilib.plugins.java/src/main/java/edu/wpi/first/wpilib/plugins/java/WPILibJavaPlugin.java:202]: Fortunately it only does expansion to the first path segment (e.g. up to the first `/`).

**Pull request merged**